compact_test.go

```diff
@@ -650,16 +723,8 @@ func TestCompaction_populateBlock(t *testing.T) {
 			c, err := NewLeveledCompactor(nil, nil, []int64{0}, nil)
 			testutil.Ok(t, err)
 
-			meta := &BlockMeta{
-				MinTime: tc.compactMinTime,
-				MaxTime: tc.compactMaxTime,
-			}
-			if meta.MaxTime == 0 {
-				meta.MaxTime = math.MaxInt64
-			}
```

Comment:
I think this will fail without 
```
if tc.compactMaxTime == 0 {
				tc.compactMaxTime = math.MaxInt64
			}
```

Reply:
I added explicit ranged to all use cases